neural_speed/models/llama/llama.h

```diff
@@ -26,18 +26,38 @@ enum llama_model {
   LLAMA_65B,
 };
 
-static const model_scratch llama_mem_req(int n_layers) {
+static const model_scratch llama_mem_req(int n_layers, float enlarge_scale = 1.0f) {
```

Comment:
could we based on batch auto set the enlarge scale ? 
Therefore costumes won't need consider this args.

Reply:
It's complicated. We need graph-alloc in the long term.  We can multiply `batch_size` after @Zhenzhong1's PR is merged in the short term.